src/utils/validators/McRequestSchema.ts

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable */
 
-import * as Joi  from 'joi';
+import * as Joi from 'joi';
 
 export const mcRequestSchema = Joi.array().items({
-  vehicleIdentifier: Joi.string().required(),
+  vehicleIdentifier: Joi.string().required().max(8),
```

Comment:
It could be that the AC needs clarifiying but it states on the story:

_No trailers are processed where the trailerId is greater than 8._

The above would apply to all vehicle types, is that still correct?